Makefile

```diff
@@ -0,0 +1,34 @@
+PLUGINS = $(sort \
+		  $(subst /, -, \
+		  $(patsubst cmd/%/, %, \
+		  $(dir \
+		  $(shell find cmd/ -type f -name '*.go')))))
+
+DOCKERFILES = $(sort \
+			  $(subst /, -, \
+			  $(patsubst cmd/%/, %, \
+			  $(dir \
+			  $(shell find cmd/ -type f -name 'Dockerfile')))))
+
+build: $(patsubst %, build-%, $(PLUGINS))
+
+build-%:
+	cd cmd/$(subst -,/,$*) && go build
```

Comment:
It would  be nicer to move binaries to some kind of _out directory already - will be needed as soon as the build is dockerized.

Reply:
We can do that once the build is dockerized. This is sufficient now and it fits docker image build process.